integration_tests/pages/page.ts

```diff
@@ -15,12 +15,15 @@ export default abstract class Page {
       })
   }
 
-  constructor(private readonly title: string) {
+  constructor(private readonly title: string, private readonly noBackLink: boolean = false) {
     this.checkOnPage()
   }
 
   checkOnPage(): void {
     cy.get('h1').contains(this.title)
+    if (this.noBackLink === false) {
+      cy.get('[data-qa=back-link]').should('exist')
```

Comment:
might be better to just check this in the unit tests instead? Think maybe method where we could check the href in the backlink in the integration tests would be good though? 🙂

Reply:
^ then again might be overkill to check the href in multiple places since we're using the inbuilt history function in js. Suppose it'd make the tests more valuable if we change that implementation later on, but we could cross that bridge when we come to it 😄

Reply:
Hey @connormaglynn , We talked about this on Friday and we don't have a way of verifying for every page in unit tests and can't test javascript so thought it might be better to just handle in integration tests